### PR TITLE
Containers don't get their own CPU

### PR DESCRIPTION
**Pull request by coderanger**
This is overly broad, in all default setup containers all share the underlying host CPUs so I wouldn't say they "have their own".

### CODE REVIEW

**Comment by bgrant0607** [comment]
CPU shares are allocated. Making this change would be confusing without a bigger change.

**Comment by coderanger** [comment]
@bgrant0607 I don't think I would call `cpu.shares` the same as "have their own CPU". A lot of the misunderstanding I find as to the difference between VMs and containers is that they don't understand how much is shared.

**Comment by kbhawkey** [comment]
@coderanger . 
It seems worthwhile to log an issue for this proposed content change.

**Comment by bgrant0607** [comment]
My suggestion is to split the OS resources and physical/virtual resources. OS resources are (mostly) isolated whereas physical/virtual resources like CPU and memory are apportioned but shared (and can be oversubscribed).

This is intended to be a concise summary rather than a deep dive, so it's not going to be fully accurate.

**Review by sftim — COMMENTED** [review]
/hold
As it stands I don't think this is the right fix @coderanger 
Let's discuss another approach

**Comment by sftim** [comment]
/retitle Containers don't get their own CPU

**Comment by bgrant0607** [comment]
Thanks @sftim. Share of CPU SGTM.

File: content/en/docs/concepts/overview/what-is-kubernetes.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Virtualization allows better utilization of resources in a physical server and a
 
 Each VM is a full machine running all the components, including its own operating system, on top of the virtualized hardware.
 
-**Container deployment era:** Containers are similar to VMs, but they have relaxed isolation properties to share the Operating System (OS) among the applications. Therefore, containers are considered lightweight. Similar to a VM, a container has its own filesystem, CPU, memory, process space, and more. As they are decoupled from the underlying infrastructure, they are portable across clouds and OS distributions.
+**Container deployment era:** Containers are similar to VMs, but they have relaxed isolation properties to share the Operating System (OS) among the applications. Therefore, containers are considered lightweight. Similar to a VM, a container has its own filesystem, memory, process space, and more. As they are decoupled from the underlying infrastructure, they are portable across clouds and OS distributions.
```
Comment (sftim) [inline content/en/docs/concepts/overview/what-is-kubernetes.md:41]:
```suggestion
**Container deployment era:** Containers are similar to VMs, but they have relaxed isolation properties to share the Operating System (OS) among the applications. Therefore, containers are considered lightweight. Similar to a VM, a container has its own filesystem, share of CPU, memory, process space, and more. As they are decoupled from the underlying infrastructure, they are portable across clouds and OS distributions.
```

**Review by sftim — COMMENTED** [review]
Thanks
/lgtm

**Comment by kbhawkey** [comment]
Thanks @coderanger .
/approve

**Pull request merged**